flutter_web_auth_2/example/ios/Podfile.lock

```diff
@@ -1,23 +1,28 @@
 PODS:
-  - Flutter (2.0.2)
-  - flutter_web_auth (0.4.1):
+  - Flutter (1.0.0)
+  - flutter_web_auth_2 (1.1.1):
```

Comment:
This seems rather odd to me, why does it not use the newest version here? Or is there something else wrongly set up or is this another Apple moment? :)

Reply:
These version numbers are based on the podspecs of the plugins which don't really mean anything. The podspec comes with the plugin and can be on whatever version and it doesn't matter.

Reply:
Here's the documentation for universal links: https://developer.apple.com/library/archive/documentation/General/Conceptual/AppSearch/UniversalLinks.html

Major differences from custom url schemes:
- The links are just normal links
- You have to host a manifest on your domain to tell iOS what links it should open in your app
- It's "more secure" probably

Reply:
Okay, thank you very much for elaborating!
Have you tested your changes and if Universal links are now properly supported?
I could imagine that this check: https://github.com/ThexXTURBOXx/flutter_web_auth_2/blob/master/flutter_web_auth_2/lib/flutter_web_auth_2.dart#L35-L44
could forbid using them, but I am unsure.

Also, one last thing before merging: How does one use Universal links in `flutter_web_auth_2` now? Could you maybe add documentation for that in the README (e.g., add a new section [in Setup](https://github.com/ThexXTURBOXx/flutter_web_auth_2/blob/master/flutter_web_auth_2/README.md#setup))?

Reply:
Universal links for OAuth redirects are working for me in my app. My concern is that I might have accidentally broken the existing method which I don't have a way to test easily.

You still need to pass a url scheme, but you just use `https`.

The setup for Universal Links might be too in depth to put in the readme. All that changes with how you use the plugin is passing `https` to the callbackUrlScheme. There is no special setup to use Universal Links with this plugin besides the normal universal link setup. I could just add that Universal Links are supported and link the Apple documentation if you want.

Reply:
Okay, passing `https` is plausible. Then, all the checks should work fine.
I hope that the old method still works. However, someone else might need to verify this.
Maybe @Knupper could verify this. But, if he does not have time, that's okay, too :)

Maybe there is a good link for the setup of Universal links? If passing `https` is the only thing that changes, I would probably only mention that and link to a setup guide for Universal links in general.